src/cli/cli_tcat.cpp

```diff
@@ -244,10 +303,13 @@ otError Tcat::Process(Arg aArgs[])
         aCommandString, &Tcat::Process<Cmd(aCommandString)> \
     }
 
-    static constexpr Command kCommands[] = {CmdEntry("start"), CmdEntry("stop"), CmdEntry("vendorid")};
+    static constexpr Command kCommands[] = {CmdEntry("advertisedid"), CmdEntry("generalid"), CmdEntry("start"),
```

Comment:
It looks like there's no documentation yet added in README_TCAT.md for the new CLI commands. This would be helpful I think. We can clarify there for example that the "General" in the name comes from the TCAT command 0x0B in the General command class. In the spec this ID is called "Device ID" so using `tcat devid` or `tcat deviceid` for this command would be also fine I think. And `advid` for the advertisedid maybe or is that too ugly?

Reply:
I do agree the command names were too long. (updated already)
Documentation is there as well.